core/src/main/java/co/aikar/commands/annotation/HelpCommand.java

```diff
@@ -26,7 +26,14 @@
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
 
+/**
+ * Sets the help command handler. Whenever the help menu is invoked, this will be called.
+ */
 @Retention(RetentionPolicy.RUNTIME)
 public @interface HelpCommand {
+    /**
+     * The command aliases of the help command.
```

Comment:
This is not correct. per above, it overrides the subcommands, not command aliases